modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -445,10 +445,16 @@ const Quirks = (editor: Editor): Quirks => {
   const normalizeSelection = () => {
     // Normalize selection for example <b>a</b><i>|a</i> becomes <b>a|</b><i>a</i>
     editor.on('keyup focusin mouseup', (e) => {
+
+      // The next three lines address TINY-4550: when there is only an image in the editor then normalization should not run
+      const selectedNode = selection.getNode();
```

Comment:
Not sure but I think I would prefer something like this to see if all of the content is selected:
```
const body = editor.getBody();
const rng = editor.selection.getRng();
const selectionIsAllContent = rng.startContainer === rng.endContainer && rng.startContainer === body && rng.startOffset === 0 && rng.endOffset === body.childNodes.length;
```
Since the 'SelectAll' command sets the explicit range to the whole contents of the editor.

Reply:
There's also the `SelectionUtils.hasAllContentsSelected` internal API that might be able to be used here instead?

Reply:
That seems to travel down so it would be true for the <p>[<img>]</p> case and in theory we would want to normalize say when the start point is before a text node.

Reply:
> Not sure but I think I would prefer something like this to see if all of the content is selected:
> 
> ```
> const body = editor.getBody();
> const rng = editor.selection.getRng();
> const selectionIsAllContent = rng.startContainer === rng.endContainer && rng.startContainer === body && rng.startOffset === 0 && rng.endOffset === body.childNodes.length;
> ```
> 
> Since the 'SelectAll' command sets the explicit range to the whole contents of the editor.

Done!